CHANGEDB.php

```diff
@@ -1003,4 +1003,6 @@
 //v23.0.00
 ++$count;
 $sql[$count][0] = '23.0.00';
-$sql[$count][1] = "";
+$sql[$count][1] = "
+ALTER TABLE `gibbonLibraryItem` ADD `cost` decimal(10,2) DEFAULT NULL AFTER `invoiceNumber`;end
+";
```

Comment:
@powertashton: Let's try to put these 3 lines into 1 single line string and see.

Reply:
Let's try indeed... hopefully it works, though it's strange it was fine the first commit and not the second.

Reply:
Hmm.. doesn't seem to be that. I suspect it has something to do with gibbon.sql rather than CHANGEDB.php as that's the only file that was changed in the commit that messed it up... Hmm 🧐

Reply:
From the diff, I don't see the `gibbon.sql` changes. I don't think that's the issue here.

There are a few other warnings / errors worth taking a look:
```
[Mon Jul  5 04:59:25 2021] PHP Warning:  SQLSTATE[42000]: Syntax error or access violation: 1091 Can't DROP 'citizenship1'; check that column/key exists in /home/runner/work/core/core/src/Database/Connection.php on line 208
[Mon Jul  5 04:59:25 2021] PHP Warning:  Undefined variable $sql in /home/runner/work/core/core/src/Database/Migrations/2021-06-04-LibraryTypes.php on line 48
```

Something must have changed to cause these warning to come up.

Reply:
I had undone the gibbon.sql change which is when the test started failing, the previous version that I've just reverted had passed. Something's definitely wrong here but I'm not sure quite what yet